Makefile

```diff
@@ -8,6 +8,7 @@
 #  * ENABLE_DHCP_GRAPH_SERVICE: Enables get-graph service to fetch minigraph files
 #    through http.
 #  * SHUTDOWN_BGP_ON_START: Sets admin-down state for all bgp peerings after restart.
+#  * ENABLE_PFC_WD_ON_START: Enable PFCWD on server-facing ports when starts.
```

Comment:
Suggest change to `ENABLE_PFC_WD_ON_START: Enable PFC Watchdog on server-facing ports when SONiC starts.` for clarity.